### PR TITLE
workbench: various shell fixes + RTS parametrization

### PR DESCRIPTION
**Pull request by mgmeier**
# Description

- `workbench` shell adjustements
- RTS parametrization vocabulary + new `forge-stress-pre` profiles

# Checklist

- [x] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated.  These may include:
  - golden tests
  - property tests
  - roundtrip tests
  - integration tests
  See [Runnings tests](https://github.com/input-output-hk/cardano-node-wiki/wiki/Running-tests) for more details
- [ ] Any changes are noted in the `CHANGELOG.md` for affected package
- [ ] The version bounds in `.cabal` files are updated
- [ ] CI passes. See note on CI.  The following CI checks are required:
  - [ ] Code is linted with `hlint`.  See `.github/workflows/check-hlint.yml` to get the `hlint` version
  - [ ] Code is formatted with `stylish-haskell`.  See `.github/workflows/stylish-haskell.yml` to get the `stylish-haskell` version
  - [ ] Code builds on Linux, MacOS and Windows for `ghc-8.10.7` and `ghc-9.2.7`
- [x] Self-reviewed the diff

### CODE REVIEW

**Review by fmaste — APPROVED** [review]
LGTM

**Comment by NadiaYvette** [comment]
The RTS parametrisation looks pretty helpful beyond just the jq fix.

**Comment by mgmeier** [comment]
> The RTS parametrisation looks pretty helpful beyond just the jq fix.

Indeed; to be expanded in the future :)

File: nix/workbench/profile/prof1-variants.jq [hunk 1]
```diff
@@ -388,7 +388,7 @@ def all_profile_variants:
     ({ extra_desc:                     "with legacy iohk-monitoring"
      , suffix:                         "iomf"
      }|
-     .node.tracing_backend           = "iohk-monitoring"
+     .node.tracing_backend            = "iohk-monitoring"
```
Comment (angerman) [inline nix/workbench/profile/prof1-variants.jq:391]: I don't follow the alignment change here?

**Review by angerman — APPROVED** [review]
A feels a bit like we are using `jq` slightly outside of its intended use case, but why not. I'm afraid I'm not going to be able to provide a throuhroigh jq review. I'm not familiar enough with jq as a scripting language 😨

File: nix/workbench/profile/prof1-variants.jq (hunk 1 above)
Comment (mgmeier) [reply nix/workbench/profile/prof1-variants.jq:391]: The line was misaligned wrt. the other node config variants.

**Pull request merged**